glib/src/param_spec.rs

```diff
@@ -1035,6 +1187,12 @@ impl ParamSpecParam {
     }
 }
 
+define_builder!(
+    ParamSpecParam,
+    ParamSpecParamBuilder,
+    param_type: crate::Type,
```

Comment:
The param type must be a concrete param type and is mandatory, so must be given in the `new()` / `builder()` function